src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -80,6 +81,9 @@ export const gravityStitchingEnvironment = (
           before: String
         ): UserAddressConnection
       }
+      extend type SearchCriteria {
+        labels: [SearchCriteriaLabel]
```

Comment:
It's harder to make these fields non-nullable later so unless we anticipate `labels: null` or `labels: [null]`, we might want to opt for the stricter return type now.

```suggestion
        labels: [SearchCriteriaLabel!]!
```